plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSessionProperties.java

```diff
@@ -252,6 +252,11 @@ public IcebergSessionProperties(
                         "Minimal retention period for remove_orphan_files procedure",
                         icebergConfig.getRemoveOrphanFilesMinRetention(),
                         false))
+                .add(booleanProperty(
+                        MERGE_MANIFESTS_ON_WRITE,
+                        "Compacting Manifest files in sync with write operation (true - Append, false - FastAppend)",
```

Comment:
Let's re-word this a bit:

> Compact manifest files when performing write operations

We can remove the true/false part since that is an Iceberg library implementation detail that end users shouldn't be concerned about.